cmd/goal/logging.go

```diff
@@ -46,7 +46,8 @@ var loggingCmd = &cobra.Command{
 	Long:  `Enable/disable and configure Algorand remote logging`,
 	Args:  validateNoPosArgsFn,
 	Run: func(cmd *cobra.Command, _ []string) {
-		cfg, err := logging.EnsureTelemetryConfig(nil, "")
+		dataDir := ensureSingleDataDir()
```

Comment:
This exits if `-d` not specified which isn't exactly what we want, right?

Reply:
duplicating logic between `goal logging` and `diagcfg telemetry` is bugging me. Can we just delete or deprecate `goal logging` and if we don't delete it let it be the slightly less capable version? Most of `goal` doesn't make sense without a data directory, so requiring a data directory here doesn't feel out of character.

Reply:
Yes - `goal logging` should be deprecated